packages/webpack/src/utils/postcss.js

```diff
@@ -28,6 +28,17 @@ export const orderPresets = {
   }
 }
 
+const postcssConfigFileWarning = (() => {
+  let executed
+  return () => {
+    if (executed) {
+      return
+    }
+    consola.warn(`Please use \`build.postcss\` in your nuxt.config.js instead of an external config file. Support for such files will be removed in Nuxt 3.`)
```

Comment:
Probably also adding a small description of why? Like because it overrides all nuxt postcss config. OR linking do the docs

Reply:
```suggestion
    consola.warn(`Please use \`build.postcss\` in your nuxt.config.js instead of an external config file. Support for such files will be removed in Nuxt 3 as they remove all defaults set by Nuxt and can cause severe problems with features like alias resolving inside your CSS.`)
```

Like that? :relaxed:

Reply:
Yep :